portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -22,27 +28,72 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import org.apache.commons.io.IOUtils;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    private List<String> comments;
-    @Override
-    public void init(){
-        comments = new ArrayList<>();
-        comments.add("This is the first comment");
-        comments.add("This is the second comment");
-        comments.add("This is the third comment");
+    public class CommentClass{
```

Comment:
Move this class def out of Servlet def?
Also, simply name it "Comment" instead of "CommentClass" as it already defined as "class"

Reply:
Will do. Resolved